rules/builds.smk

```diff
@@ -508,9 +508,9 @@ rule colors:
         color_schemes = config["files"]["color_schemes"],
         metadata = _get_metadata_by_wildcards
     output:
-        colors = "config/colors_{region}.tsv"
+        colors = "config/colors_{build_name}.tsv"
```

Comment:
I think this is the right time to change this output to `"results/{build_name}/colors.tsv"`

Reply:
done